### DISCUSSION TITLE
Um site pessoal simplório demais, pode causar má impressão?

### DISCUSSION THREAD

**Discussion by fabiomirandaa** [frontendbr/forum#462] [Discussão]
Quando um desenvolvedor se dispõe a ter um site, se ele não cuidar bem(deixando bugs), ou esse site tiver um layout ruim, pode causar um preconceito na hora de uma possível contratação etc?

**Comment by oguilhermelessa** [comment]
Com certeza! Assim como todos os outros tipos de profissionais que vendem seus serviços através de "portfólios", o desenvolvedor não está fora desse julgamento. Costumo dizer que o site tem que ser uma extensão da sua "empresa" ou a melhor, a melhor forma de mostrar suas habilidades.

**Comment by Cleysonlb** [comment]
@fabiomirandaa um site com layout ruim e bugs vai sim causar uma má impressão, agora, um site simples mas bem desenvolvido não!
Cara parta do simples, se não conhece tanto de design pega um framework UI legal, se não tem tempo ou conhecimento pra fazer algo muito grande, faz algo simples, html puro e poem la no ar!

**Comment by diessica** [comment]
Site simples não é site mal feito, necessariamente... E nem todo programador é designer, certo?

Eu definitivamente não teria preconceito. Agora, um site mal desenvolvido (nisso várias coisas podem ser analisadas: decisões exageradas de tecnologia, desorganização, más práticas...) abre sim portas para julgamentos, mas nada que excluiria você de um processo seletivo. Um código ruim significa algo sim, mas não define o trabalho de ninguém.

Mesmo assim, é sempre legal ter carinho e cuidado pelo nosso trabalho. Um site bem feito transparece bastante isso, o que é positivo na hora de uma contratação.

**Comment by kvnol** [comment]
Eu sempre parti do princípio que menos é mais. Amo o minimalismo e vou defendê-lo.

E, como a @diessica disse, "site simples não é site mal feito".

**Comment by felipefialho** [comment]
O design (e tecnologia também) deve servir ao CONTEÚDO e ser um facilitador de acesso a ele.

Por isso, como sempre, concordo com a @diessica. Um site simples jamais seria um problema, agora um site mal desenvolvido, mal pensado e com conteúdo ruim... com certeza seria.

**Comment by dionarodrigues** [comment]
Em um aspecto geral, como o site/portfolio pode ser uma das variáveis analisada ao se concorrer a uma vaga de emprego, se ele estiver mal feito penso que pode sim prejudicar na seleção - já que ele irá refletir seu trabalho como um todo (se você é organizado, caprichoso, se pensa no usuário, se tem bom senso...).

E, apesar do famoso ditado "Casa de ferreiro, espeto de pau", eu não apostaria num site bugado para fazer meu marketing pessoal.

Agora, quanto à questão da simplicidade, é preciso levar em consideração o que você quer dizer com simplicidade: não usar tecnologias de ponta?; não aderir às tendências de design para se criar a interface?; não utilizar um menu hamburguer? O que seria esse simples?

Também em um aspecto geral, penso que o site não precisa estar no topo dos melhores do mundo, utilizando as tecnologias mais modernas, porém, ele precisa transmitir sua mensagem de modo claro e objetivo, em uma interface que funcione e que guie o olhar do usuário pelos conteúdos mais importantes. Precisa ter uma ótima apresentação sobre você, seus trabalhos, suas ideias para solucionar os problemas que precisou resolver... E também precisa mostrar seus dados de contato, claro. Evitar erros de português.
